internal/commands/result.go

```diff
@@ -52,6 +52,11 @@ const (
 	lowCx                     = "LOW"
 	mediumCx                  = "MEDIUM"
 	highCx                    = "HIGH"
+	tableResultsFormat        = "              | %-10s   %4d   %6d   %4d   %4d   %-9s  |\n"
+	stringTableResultsFormat  = "              | %-10s   %4s   %6s   %4s   %4s   %5s      |\n"
+	TableTitleFormat          = "              | %-11s   %4s    %6s  %4s   %4s   %6s    |\n"
+	twoNewLines               = "\n\n"
+	tableLine                 = "              ---------------------------------------------------------     "
 	codeBashingKey            = "cb-url"
```

Comment:
Why put those fields as constants if they are only related to printResultsSummaryTable function?

Reply:
some of them are used from more than one function, and it is order and clear to see the table format together - so when you want to change the format - you see all the line formaters together